### PR TITLE
release `0.11.9`

### PR DESCRIPTION
**Pull request by nleroy917**
Hot release for pephub due to database changes:

- [x] bump version
- [x] update changelog

### CODE REVIEW

**Comment by nleroy917** [comment]
Works locally, deployed successfully -- ready when you are @khoroshevskyi

File: pephub/dependencies.py [hunk 1]
```diff
@@ -441,13 +399,9 @@ def get_namespace_info(
...
-            f"Namespace '{namespace}' does not exist in database. Did you spell it correctly?",
-        )
-    else:
-        yield namespace
+@cached(TTLCache(maxsize=100, ttl=5 * 60))
+def get_pepdb_namespace_info(limit: int = 10) -> ListOfNamespaceInfo:
+    """
+    Get the information on the biggest namespaces in the database.
+    """
+    return agent.namespace.info(limit=limit)
```
Comment (nleroy917) [inline pephub/dependencies.py:407]: Should we increase the cache time for this?

File: pephub/routers/api/v1/helpers.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    SAMPLE_NAME_ATTR,
+    SAMPLE_RAW_DICT_KEY,
+    SAMPLE_TABLE_INDEX_KEY,
+    SUBSAMPLE_RAW_LIST_KEY,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def verify_updated_project(updated_project) -> peppy.Project:
```
Comment (nleroy917) [inline pephub/routers/api/v1/helpers.py:19]: Type annotation and doc string?

File: pephub/routers/api/v1/helpers.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    except Exception as _:
+        raise HTTPException(
+            status_code=400,
+            detail="Could not validate PEP. Please check your PEP and try again.",
+        )
+
+    return new_project
+
+
+async def check_sample_names(sample_list: list, sample_table_index_col: str) -> None:
```
Comment (nleroy917) [inline pephub/routers/api/v1/helpers.py:76]: Same here, type annotation and doc strings?

File: pephub/routers/api/v1/project.py [hunk 4]
```diff
@@ -1,58 +1,49 @@
...
+from peppy.const import SAMPLE_DF_KEY, SAMPLE_RAW_DICT_KEY
 
-from dotenv import load_dotenv
-
-
-from ...models import ProjectOptional, ProjectRawModel, ForkRequest
-from ....helpers import zip_conv_result, zip_pep
+from ....const import SAMPLE_CONVERSION_FUNCTIONS
 from ....dependencies import (
+    DEFAULT_TAG,
```
Comment (nleroy917) [inline pephub/routers/api/v1/project.py:34]: import from `const.py`?

File: pephub/routers/api/v1/project.py [hunk 5]
```diff
@@ -104,56 +94,11 @@ async def get_a_pep(
...
-
-    return dict(
-        **proj_dict,
-        **proj_annotation_dict,
-        samples=samples,
-        sample_table_indx=sample_table_index,
-        sample_attributes=sample_attributes,
-    )
+    except Exception:
+        raise HTTPException(500, "Unexpected project error!")
```
Comment (nleroy917) [inline pephub/routers/api/v1/project.py:101]: Should we try to parse/anticipate any errors?

File: requirements/requirements-all.txt [hunk 6]
```diff
@@ -1,7 +1,7 @@
 fastapi>=0.108.0
 psycopg>=3.1.15
-# pepdbagent @ git+https://github.com/pepkit/pepdbagent.git@dev#egg=pepdbagent
-pepdbagent>=0.8.0
+pepdbagent @ git+https://github.com/pepkit/pepdbagent.git@dev#egg=pepdbagent
+# pepdbagent>=0.9.0
```
Comment (nleroy917) [inline requirements/requirements-all.txt:4]: Need to point at deployed `pepdbagent` on pypi

File: web/src/components/modals/fork-pep.tsx [hunk 7]
```diff
@@ -22,6 +23,41 @@ interface ForkProjectInputs {
...
+  } else if (nameError == 'empty' && tagError == 'empty') {
+    msg = 'Project Name and Tag must not be empty.';
+  } else if (nameError == 'invalid' && tagError == 'empty') {
+    msg = "Project Name must contain only alphanumeric characters, '-', or '_' and Tag must not be empty.";
+  } else if (!nameError && tagError == 'empty') {
+    msg = 'Project Tag must not be empty.';
+  } else if (!nameError && tagError == 'invalid') {
+    msg = "Project Tag must contain only alphanumeric characters, '-', or '_'.";
+  }
+
```
Comment (nleroy917) [inline web/src/components/modals/fork-pep.tsx:53]: I've seen this in a lot of places... wondering two things:

1. Can this be abstracted to a function, and
2. All these `if-else`'s seem like an anti-pattern. Can it not be a `switch-case`?

**Review by nleroy917 — COMMENTED** [review]
This looks *ok* to me, but there are things I think should be addressed. Due to the need to get it out, its probably fine for now, and we could open a "cleanup" issue for things that I addressed in the review.

File: web/src/components/modals/fork-pep.tsx (hunk 7 above)
Comment (sanghoonio) [reply web/src/components/modals/fork-pep.tsx:53]: I wanted to make it a function, but wasn't sure where to put it

Comment (nleroy917) [reply web/src/components/modals/fork-pep.tsx:53]: Yeah good q. I think I have a `utils.ts` file somewhere I put small little helpers like these

Comment (nleroy917) [reply web/src/components/modals/fork-pep.tsx:53]: yeah. `web/src/utils/[file].ts` stick in one of those that makes sense or just make another 🙂

**Review by khoroshevskyi — APPROVED** [review]

**Pull request merged**